docs/manuals/deployment/linux/redhat.md

```diff
@@ -0,0 +1,190 @@
+The name of the variable where the Web application is stored is used in the _uWSGI_ configuration:
+this allows the Web server to load the Web application:
+```
+from taipy import Gui
+
+gui_service = Gui(page="# Getting started with *Taipy*")
+web_app = gui_service.run(debug=False, run_server=False)
+```
+In our example, we store this application in the variable _web_app_ (see line 3)
+
+Make sure you upload this code on your targeted machine and install your dependencies with _pip_.
+
+!!! important
+    The entrypoint filename and the app variable name are important for the proper configuration of
+    the _uWSGI_ Web application server. Please, keep them as is or adapt the configuration.
+
+
+## uWSGI application server
+
+To expose your application over the Internet, you must use _uWSGI_ instead of Flask as the application server.
+You would then leverage Nginx to expose the application.
+
+_uWSGI_ can be started manually. But, generally, it's better to start the application automatically when the machine
+starts. To order to do that, you should use [Systemd](https://systemd.io/) which is installed by default on _Ubuntu_.
+
+Beside your application code, run the following command to generate an adapted file for _Systemd_:
+```
+echo """
+[Unit]
+Description=App
+After=syslog.target
+
+[Service]
+ExecStart=uwsgi --http 127.0.0.1:5000 --gevent 1000 --http-websockets --module app:web_app
+WorkingDirectory=`pwd`
+Restart=always
+KillSignal=SIGQUIT
+Type=notify
+StandardError=syslog
+NotifyAccess=all
+User=`whoami`
+
+[Install]
+WantedBy=multi-user.target
+""" > app.uwsgi.service
+```
+Then transfer this file in the correct folder by doing:
+```
+sudo mv app.uwsgi.service /etc/systemd/system/app.uwsgi.service
+```
+
+Now, you can start your application automatically at the startup time of your machine by doing:
+```
+sudo restorecon /etc/systemd/system/app.uwsgi.service
+sudo systemctl daemon-reload
+sudo systemctl start app.uwsgi.service
+sudo systemctl enable app.uwsgi.service
+```
+
+The application is now running locally but is not accessible yet from the Internet.
+
+
+## Exposing to the Internet
+
+To be able to access to your application from the Internet, you should use _Nginx_.
+Replace the content of `/etc/nginx/nginx.conf` by the [following](./nginx.conf) or:
+```
+sudo wget https://docs.taipy.io/manuals/deployment/linux/redhat/nginx.conf -O /etc/nginx/nginx.conf
+```
+
+Allow the communication between _Nginx_ and _uWSGI_:
+```
+sudo setsebool -P httpd_can_network_connect 1
+```
+The restart _Nginx_:
+```
+sudo systemctl restart nginx
+```
+
+!!! Note
+    This configuration is only for HTTP. If you need an HTTPS connection, follow the [Nginx documentation](https://nginx.org/en/docs/http/configuring_https_servers.html).
+
+
+## Open firewall
+
+Your application is ready to receive traffic from the Internet but your firewall still block the communication.
+Allow the connection on the _http_ port that is _80_:
```

Comment:
"Allow the connection on the _http_ port that is _80_"
->
"Open the _http_ port that is (i.e. port _80_)"